rust/src/dataset.rs

```diff
@@ -553,42 +553,102 @@ impl Dataset {
         Ok(())
     }
 
-    /// Create a new version of [`Dataset`] from a collection of fragments.
+    /// Commit changes to the dataset
```

Comment:
Since it might not be obvious, we should say that to create a new table, use the `Overwrite` operation.

Reply:
I added a note to this effect:

```
    /// All operations except Overwrite will fail if the dataset does not already exist.
```